tools/coqworkmgr/dune

```
@@ -1,7 +1,7 @@
 (library
  (name coqworkmgrapi)
```

If we do
```suggestion
 (name coqworkmgrlib)
```
don't rename the coqworkmgrApi.ml and don't change the public_name I think there is no breakage (because `wrapped false`) and still fixes the bug?

---

That's what I wanted to do at first, but this changes the list of observable `coq-core` libs in `ocamlfind list` so I was scared of it for a minor release.

But in the end it seems that this solution may be the best after all, but we should check with 8.17 RMs.

SerAPI's `main` should not need overlay after https://github.com/ejgallego/coq-serapi/pull/340 anyways.

---

Why do the coq-core libs change if we keep the public_name the same? Maybe I'm misunderstanding something.

---

Indeed I forgot that there is indeed nothing that makes the wrapper / lib name be a subset of the public name. Let's go that way.